lib/beetle_etl/steps/step.rb

```diff
@@ -37,5 +35,12 @@ def stage_table_name(table_name = nil)
       BeetleETL::Naming.stage_table_name(external_source, table_name || @table_name)
     end
 
+    def mappings_table_name(table_name = nil)
```

Comment:
Why do you redefine those 2 functions here? Instead of just calling the one from the naming module directly?

Reply:
Because of the `@table_name` that is initialized by the classes that inherit from this one.